web/viewer.html

```diff
@@ -137,8 +137,13 @@
             <input type="checkbox" id="findMatchCase" class="toolbarField" tabindex="95">
             <label for="findMatchCase" class="toolbarLabel" data-l10n-id="find_match_case_label">Match case</label>
           </div>
+          <div id="findbarOptionsThreeContainer">
+            <input type="checkbox" id="findMatchDiacritics" class="toolbarField" tabindex="96">
+            <label for="findMatchDiacritics" class="toolbarLabel" data-l10n-id="find_match_diacritics_label">Match diacritics</label>
```

Comment:
`Match diacritics` -> `Match Diacritics` such that the capitalization matches the Firefox findbar.

Also, please remember to add `find_match_diacritics_label=Match Diacritics` *after* this line as well (such that it can actually be localized): https://github.com/mozilla/pdf.js/blob/fd82adccfa409f9b2cc72ad2cb8555bd2aa68b36/l10n/en-US/viewer.properties#L170

Reply:
About the case on `Diacritics`, I just followed what we already have in this file (lower case for the second word).
I just checked and in firefox, we've "Highlight All", "Match Case" and "Whole Words".
We can fix in a follow-up.